modules/account-lib/src/coin/hbar/tokenTransferBuilder.ts

```diff
@@ -0,0 +1,104 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import * as Long from 'long';
+import { TokenId } from '@hashgraph/sdk';
+import { proto } from '../../../resources/hbar/protobuf/hedera';
+import { BuildTransactionError, InvalidParameterValueError } from '@bitgo/sdk-core';
+import { TransferBuilder } from './transferBuilder';
+import { Transaction } from './transaction';
+import { getHbarTokenFromTokenName, isValidToken, stringifyAccountId, stringifyTokenId } from './utils';
+
+export class TokenTransferBuilder extends TransferBuilder {
```

Comment:
After our conversation, I still think it'd be in bad form to have `TokenTransferBuilder` extend `TransferBuilder`. I think coupling it too tightly to the TransferBuilder may just be a headache later on.
Maybe instead of `A` we can have `B`
```
A. TransactionBuilder -> TransferBuilder -> TokenTransferBuilder

                                            CoinTransferBuilder
B. TransactionBuilder -> TransferBuilder <
                                            TokenTransferBuilder
```

btw this change doesn't have to be apart of this PR, I just wanted to share my thoughts

Reply:
I'll think about it, since my current concern is CoinTransferBuilder will require a change in microservice as well due to adding coinTransferBuilder. 
And let say we want to migrate to using diff class for coin and token, then I kinda want to change to support multi-recipients as well as the flow to update into new class will be in both account-lib and ms for the native coin

and if so, then it will be just:

                                       CoinTransferBuilder
TransactionBuilder <
                                       TokenTransferBuilder

which is my plan if tokenTransfer can be finished and there's still time to update

Reply:
Actually, I can extend TransactionBuilder with different format from TransferBuilder (like using `send` instead of `to` and `amount`) where this can support multi recipients but currently just used to handle 1 recipient will be fine as well.